config/coolstore.config.js

```diff
@@ -17,4 +19,6 @@ if (process.env.SECURE_COOLSTORE_GW_ENDPOINT != null) {
     config.SECURE_API_ENDPOINT = process.env.SECURE_COOLSTORE_GW_SERVICE + '-' + process.env.OPENSHIFT_BUILD_NAMESPACE;
 }
 
+config.PRODUCT_REFRESH_INTERVAL = get('PRODUCT_REFRESH_INTERVAL').default(5000).asIntPositive()
```

Comment:
If the UI should refresh more frequently, you can use this to lower the delay between polls against the backend. It's a millisecond value that defaults to 5000ms by default.